### DISCUSSION TITLE
LoadH5Seurat error after updating signac package

### DISCUSSION THREAD

**Discussion by Crystalli2018** [satijalab/seurat#4448] [Q&A] [answered]
Dear development team,
I recently updated the signac package to v1.2.0, but when I re-run a script using Seurat v4.0:
reference <- LoadH5Seurat("pbmc_multimodal.h5seurat")

I got the following error which didn't exist before the signac update:
------------------
...
Adding cell embeddings for wnn.umap
Adding miscellaneous information for wnn.umap
Adding graph wknn
Error in validObject(.Object) :
  invalid class "Graph" object: superclass "Mnumeric" not defined in the environment of the object's class
Calls: LoadH5Seurat ... initialize -> callNextMethod -> .nextMethod -> validObject
Execution halted
--------------------

I wonder if the Signac v1.2.0 changed something shared with Seurat package. Please help. Thanks.

**Comment by coopershawna** [comment]
Error in validObject(.Object) :
invalid class "Graph" object: superclass "Mnumeric" not defined in the environment of the object's class

Got the same error when running "find neighbors".

**Reply by coopershawna** [reply]
also after upgrading signac

**Comment by timoast** [comment, marked as answer]
See https://github.com/satijalab/seurat/issues/4436 (nothing to do with Signac)

**Reply by coopershawna** [reply]
Thank you Dr. Stuart!
